test/query.spec.js

```diff
@@ -57,19 +57,19 @@ describe('Query', () => {
     dht.switch.dial = (peer, callback) => callback()
 
     let i = 0
-    const query = (p, cb) => {
+    const query = async (p) => {
```

Comment:
why `async` here?

Reply:
this is the `queryFunc` which is an async fn. even if `await syncFn()` works fine, something like `consumePromise(syncFn())` may explode if it does not get a promise